### PR TITLE
Dp 22592/add new global footer

### PR DESCRIPTION
**Pull request by clairesunstudio**
Changed:
  - project: Patternlab
    component: Footer, FooterWithColumns
    description: Renamed Footer as FooterWithColumns, add new footer component as Footer. (#1629)
    issue: DP-22592
    impact: Minor
Added: 
  - project: Patternlab, React, Assets
    component: SocialLinks
    description: Added inverted and `c-primary` and `c-primary-alt` variants to SocialLinks, updated Footer social links to use q1-i8 color. (#1629)
    issue: DP-22592
    impact: Minor
  - project: Assets
    component: StickyNav, Details
    description: Refactored StickyNav layout styles into a reusable mixin, to use to align footer content. (#1629)
    issue: DP-22592
    impact: Minor

Mayflower UAT: https://mayflower.digital.mass.gov/patternlab/b/DP-22592/add-new-global-footer/?p=organisms-footer
Tugboat UAT: https://pr1525-f22zlri7etuqyy4aagjf7s4fo39wf6jk.tugboatqa.com/

### CODE REVIEW

**Comment by clairesunstudio** [comment]
The react backstop failures are not related to this PR. I will create a separate ticket to look into those.

Screen Shot 2022-05-19 at 9 14 41 AM

File: packages/assets/scss/00-base/mixins/_ma-layout-left-col.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  margin-right: $layout-left-col-gutter;
+
+  @media ($bp-x-large-min) {
+    width: $layout-left-col-width-x-largin;
+  }
+
+  @media ($bp-medium-max) {
+    width: 100%;
+  }
+}
```
Comment (arthurbaghdas) [inline packages/assets/scss/00-base/mixins/_ma-layout-left-col.scss:20]: Please add a new line at the end of the file.

**Comment by arthurbaghdas** [comment]
This looks good to me, but can you resolve the conflicts? @clairesunstudio

**Review by arthurbaghdas — APPROVED** [review]
Can be merged, once conflicts are solved.

**Pull request merged**